### ISSUE TITLE
Exception when applying filters or analysis on columns of dtype=datetime64[ns, tzutc()]

### ISSUE THREAD

**Issue by lcorrigan** [man-group/dtale#602]
Issue: When applying a filter, a custom filter query or simply 'Describe()' to a  column of datetime64ns, I get errors regarding comparison of tz-naive and tz-aware datetime-like objects. 
I'm using the following on python 3.8.10:
dtale                     1.61.1
numpy                     1.21.4
pandas                    1.3.4

![image](https://user-images.githubusercontent.com/12451259/142875561-92f1e748-a311-4ab7-83f2-67946257f8a9.png)

**Comment by lcorrigan** [comment]
I don't usually use this function: dateutil.parser.isoparse() but it seems my introducing it to parse some dates from JIRA's API is the reason my normally happy Datetime columns were giving me trouble today. Sorry - I had just assumed something had changed in Dtale since I last used it. 
If I cast the columns to .astype('datetime64[ns]'), everything is fine. 
Either way, I'm not sure if it's a bug that it couldn't handle the timezone or not, so I'll leave this open for the Dtale team to decide. Thanks.

**Comment by aschonfeld** [comment]
@lcorrigan thanks for the submission! I'll take a look

**Comment by aschonfeld** [comment]
@lcorrigan I'm having trouble trying to reproduce this error. I've been trying to use the following dataframe, but to no avail:

```
import pandas as pd
from dateutil.parser import isoparse

pd.DataFrame(dict(
    x=['a', 'b', 'c'],
    y=[isoparse(x) for x in ['2021-01-01', '2021-01-02', '2021-01-03']]
))
```

**Comment by lcorrigan** [comment]
Hi @aschonfeld,
When I use the JIRA API (mostly taken from here: [[and](https://levelup.gitconnected.com/jira-api-with-python-and-pandas-c1226fd41219)](https://levelup.gitconnected.com/jira-api-with-python-and-pandas-c1226fd41219) to normalize the returned json, the dates are in this format (example): 2021-11-22T21:58:37.000+0000.  I've attached a CSV file of two columns of this, one has some None's in it, as returned by the API for that column.  Hope this helps.  
[dates_for_andrew.csv](https://github.com/man-group/dtale/files/7603021/dates_for_andrew.csv)

**Comment by aschonfeld** [comment]
@lcorrigan sorry for the wait.  I've started digging into this and I was able to recreate it.  I've found a post on stackoverflow that looks promising: https://stackoverflow.com/a/34687479/12616360

I think I might just force date comparisons to be timezone-naive.

**Comment by aschonfeld** [comment]
fix added in v2.1.0
